auth_saml_ol_groups/models/auth_saml_operator.py

```diff
@@ -14,19 +14,18 @@ def operators(self):
         return ('contains', 'equals')
 
     def contains(self, attrs, mapping):
-        matching_value = ''
-        for k in attrs:
-            if isinstance(k, tuple) and mapping.saml_attribute in k:
-                for matching_value in attrs[k]:
-                    if mapping.value in matching_value:
-                        return True
+        values = attrs.get(mapping.saml_attribute)
+        if values:
+            for value in values:
+                #  Does this level even exist in the reply?
```

Comment:
Does it exist? 
a few rows down, in "equals", there seems to be no such question?